terraform/protoc-gen-terraform-teleport.yaml

```diff
@@ -418,6 +455,10 @@ validators:
     - UseVersionBetween(2,2)
   SAMLConnectorV2.Spec:
     - UseAnyOfValidator("entity_descriptor", "entity_descriptor_url")
+  ServerV2.Version:
+    - UseVersionBetween(2,2)
+  ServerV2.SubKind:
+    - UseValueIn([]string{"openssh", "openssh-ec2-ice"})
```

Comment:
:thinking: Do we need to use `[]string`?
For the `UseAnyOfValidator` we only need to pass the actual strings.